layers/+email/mu4e/packages.el

```diff
@@ -14,7 +14,7 @@
         (mu4e :location site)
         mu4e-alert
         mu4e-maildirs-extension
-        org
```

Comment:
@JorisE I think you should not remove org here, otherwise org-mu4e will not load.